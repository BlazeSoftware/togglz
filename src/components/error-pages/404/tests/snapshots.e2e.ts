import snapIt from '../../../../test/snap-it';

describe('404', () => {
  snapIt('page-not-found')('renders correctly', '<page-not-found></page-not-found>');
});
