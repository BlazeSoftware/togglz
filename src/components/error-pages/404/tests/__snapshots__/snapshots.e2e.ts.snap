// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `"<page-not-found class=\\"hydrated\\"><div class=\\"o-container o-container--small u-window-box-xlarge u-centered\\"><stencil-route-title class=\\"hydrated\\"></stencil-route-title><h2 class=\\"c-heading\\">That page does not exist.</h2><div><a class=\\"c-link\\" href=\\"/\\">Go back to the home page</a></div></div></page-not-found>"`;
